lib/mayaUsd/fileio/translators/translatorXformable.cpp

```diff
@@ -397,9 +399,9 @@ static bool _pushUSDXformToMayaXform(
 
         timeArray.setLength(timeCodes.size());
     } else {
-        // If there were no time samples, we'll just use the default time and
+        // If there were no time samples, pick the first available sample or default and
         // leave the MTimeArray empty.
-        timeCodes.push_back(UsdTimeCode::Default());
+        timeCodes.push_back(UsdTimeCode::EarliestTime());
```

Comment:
This is the actual fix. _pushUSDXformOpToMayaXform() uses EarliestTime. (As do other translators, like for meshes.)